docs/manual/installation/system-requirements.de.md

```diff
@@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
 [`fastcgi_finish_request()`](https://www.php.net/manual/en/function.fastcgi-finish-request.php) kann Contao 
 Hintergrundaufgaben (wie die Indizierung des Seiteninhalts) ausführen, ohne dass der Browser auf die Antwort wartet.
 
+### Hosting-Konfiguration
+
+In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `/web` der Installation. Setze das 
```

Comment:
@fritzmg Habt ihr eine Konvention festgelegt, wie Ordnernamen ausgezeichnet werden sollen? Im Manual findet man diese beiden Schreibweisen:
* `/web`
* `web`

Ich persönlich würde ja überall die unter Unix/Linux übliche Notation `web/` verwenden, aber das ist auch nur meine persönliche Präferenz.

Reply:
Sollten wir noch festlegen. In der dev doku ist es `web/` (evt. aber auch noch nicht durchgängig).

Reply:
okidoki